guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode.adoc

```diff
@@ -0,0 +1,33 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode_{context}"]
+= Setting an alternative directory for remote execution jobs in pull mode
+
+By default, {Project} uses the `/run` directory on host for remote execution jobs in pull mode.
+
+If the `/run` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
+Use this procedure set a new directory for executing remote execution jobs in pull mode.
+
+.Procedure
+. On your host, create a new directory:
```

Comment:
Considering that all the steps are to be performed on the same host, I suggest splitting the information to create a brief introduction to the steps:
```suggestion
On your host, perform these steps:

. Create a new directory:
```

Reply:
Applied.